index.html

```diff
@@ -1,12 +1,216 @@
+            <a class="text-20 custom-white" href="#">Join</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">Sponsor</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">News</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">My page</a>
+          </li>
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="#">Logout</a>
+          </li>
+        </ul>
+      </div>
+      <!-- top bar -->
+      <div class="top-bar flex-row">
+        <a href="#"><img src="./assets/img/facebook.png" /></a>
+        <a href="#"><img src="./assets/img/twitter.png" /></a>
+        <a href="#" class="text-12-LH-14">English</a>
+        <a href="#" class="text-12-LH-14">My Page</a>
+        <a href="#" class="text-12-LH-14">Logout</a>
+      </div>
+      <!-- Menu items -->
+      <nav class="">
+        <!-- Hamburger-->
+        <img
+          id="hamburger"
+          class="hamburger"
+          src="./assets/img/mobile-menu-1.png"
+          alt=""
+        />
+        <!-- LOGO -->
+        <a href="index.html">
+          <img class="logo" src="./assets/img/logo.png" alt="" />
+        </a>
+        <ul>
+          <a href="about.html" class="text-14-bold">About</a>
+          <a href="#" class="text-14-bold">Program</a>
+          <a href="#" class="text-14-bold">Join</a>
+          <a href="#" class="text-14-bold">Sponsor</a>
+          <a href="#" class="text-14-bold">News</a>
+          <!-- Action button -->
+          <a href="#" class="action-btn-1 text-14-bold">WD Campaign</a>
+        </ul>
+      </nav>
+    </header>
+    <!-- Hero section -->
+    <section class="hero-container flex-column">
+      <div class="hero flex-column">
+        <div class="hero-title">
+        </div>
+        <p class="sub-theme flex-row text-12 big-text-20 custom-gray-2">
+          A joyful celebration believing in the value of openness and sharing,
+          creating a positive change with people from over 80 countries is
+          taking place in October, in Korea.
+        </p>
+        <div class="date-venue flex-column">
+          <h2 class="custom-gray-3 text-20-bold big-text-42">2023.10.19(THU) ~ 20(FRI)</h2>
+          <p class="custom-gray-4 text-13 big-text-27">
+            @ National Museum of Karea, Art Center Nabi and more
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- Program section-->
+    <section class="program-container">
+      <div class="program flex-column">
+        <div class="title flex-column">
+          <h4 class="custom-white text-20 flex-row">Main Program</h4>
+          <div class="title-underline"></div>
+        </div>
+        <div class="program-details">
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/lecture-icon.png" />
```

Comment:
[REQUIRED] Please include the alt attribute and a value in all image tags that are not just meant for styling or design. But add an empty string as the value if the image is meant for styling and beautification.